src/pages/HomePage.jsx

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState } from "react";
+import { getAllContacts } from "../api/contacts";
+import TableRow from "../components/TableRow";
+import { useNavigate } from "react-router-dom";
+
+function HomePage() {
+  const [contacts, setContatcs] = useState([]);
+  const navigate = useNavigate();
+  useEffect(() => {
+    getAllContacts().then(({ data }) => {
+      setContatcs(data);
+      console.log(data);
+    });
+  }, []);
+  function newContactHandler(e) {
+    e.preventDefault();
+    navigate(`/update?id=${Date.now()}`);
+  }
+  return (
```

Comment:
Please add empty line breaks between statement that do different things 
If you write everything without empty lines it looks like spaghetti code and it is really hard to read it